enterprise-suite/es-monitor-api/static-rules.yml

```diff
@@ -1,17 +1,24 @@
+# node-exporter metrics
 - record: node_cpu_percent
-  expr: 100 - avg by (instance) (irate(node_cpu{mode="idle"}[5m]) * 100)
+  expr: 100 - avg by (es_workload, node_ip) (irate(node_cpu{mode="idle"}[5m]) * 100)
 
 - record: node_memory_usable_percent
-  expr: avg by (instance) (node_memory_MemAvailable / node_memory_MemTotal) * 100
+  expr: avg by (es_workload, node_ip) (node_memory_MemAvailable / node_memory_MemTotal) * 100
 
 - record: node_filesystem_free_percent
-  expr: avg by (device, instance) ((node_filesystem_free{device=~"/.*"} / node_filesystem_size * 100) and (node_filesystem_size > 0))
+  expr: avg by (es_workload, device, node_ip) ((node_filesystem_free{device=~"/.*"} / node_filesystem_size * 100) and (node_filesystem_size > 0))
 
 - record: node_network_error_rate
-  expr: sum by(instance) (rate(node_network_transmit_errs[5m]) + rate(node_network_receive_errs[5m]) + rate( node_network_receive_drop[5m]) + rate(node_network_transmit_drop[5m]))
+  expr: sum by(es_workload, node_ip) (rate(node_network_transmit_errs[5m]) + rate(node_network_receive_errs[5m]) + rate( node_network_receive_drop[5m]) + rate(node_network_transmit_drop[5m]))
+
+# include all nodes in the pod info, so the UI can query a single metric
+- record: kube_pod_info
+  expr: kube_node_info
+  labels:
+    pod: cluster-node
```

Comment:
That's @alanngai's call isn't it? By setting `pod` to the same, the cluster map groups all the nodes together, which I thought is what we wanted. The time series themselves remain distinct as all the labels are copied over, including the unique `node_ip`. Note I don't think you can dynamically relabel here, so I wouldn't be able to set it to `node_ip` either.

Reply:
If the idea is to get a call by Alan, just be sure to show something with multiple nodes so this design is clear. I don't think is safe to say anyone knows what they want here.

The current Map goes the other way: it shows the pods that are unhealthy in the inner circle, and a rollup health in the containing workload. This is a reimagining of that design, so some extent.

For my part, if this is in the UI, it should be able to show the node with the issue somewhere, and not just be a big red button "something in the cluster is bad". Maybe this way can do that, but one layer deeper? I'm guessing. Screen shots for a multi-node cluster where one node is alerting would help inform the decision, otherwise leaving it to the imagination.